stormpath/stormpath-sdk-angularjs
# Fixed bug that displays social login titles without any providers

Fixes a bug that made `ng-show` evaluate an empty array to `false` and display text related to social login, even though no social login providers were active.
## How to verify
1. Checkout this branch
2. Copy the `dist` folder to your AngularJS app
3. Make sure you are using the latest version of AngularJS
4. Verify that the login and register forms **doesn't** display anything related to social login when no social providers are activated
5. Add a Social Directory to your Stormpath Application
6. Verify that the login and register forms displays buttons for the activated social provider
## Screenshots

<img width="489" alt="screen shot 2015-11-20 at 17 16 04" src="https://cloud.githubusercontent.com/assets/721091/11304917/764d6e5c-8faa-11e5-857f-41c1101c2ec6.png">
